### ISSUE TITLE
Contact-us page form label issues

### ISSUE THREAD

**Issue by TheKolega** [ALASdoo/alasdoo_com_v2#173]
We have several `missing-label` and `multiple-labels` issues on the same elements, this should be fixed by scoping and aria-labeledby tags

Desc from Wave:
An <input> (except types of image, submit, reset, button, or hidden), <select>, or <textarea> does not have a properly associated label. A properly associated label is:

-     a non-hidden <label> element with a for attribute value that is equal to the id of a unique form control
-     a <label> element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute
-     a non-empty title attribute, or
-     a non-empty aria-labelledby attribute

**Comment by Tajana-Somogyi** [comment]
Closing this one, #189 has different issues
